tools/serverpod_cli/pubspec.yaml

```diff
@@ -19,7 +19,7 @@ dependencies:
   recase: ^4.0.0
   super_string: ^1.0.3
   watcher: ^1.0.0
-  analyzer: ^3.3.1
```

Comment:
Is this change required within the scope of this PR?

Reply:
completely my bad. I did changes in the same branch. I will revert them.